.idea/lessoner_project.iml

```diff
@@ -183,4 +183,9 @@
       </list>
     </option>
   </component>
+  <component name="RakeTasksCache">
+    <option name="myRootTask">
+      <RakeTaskImpl id="rake" />
+    </option>
+  </component>
```

Comment:
You should not upload this folder to the git. Please setup .idea folder for gitignore and update your PR.